price_parser/parser.py

```diff
@@ -207,7 +207,7 @@ def extract_price_text(price: str) -> Optional[str]:
 ([.,€])      # decimal separator
 (?:          # 1,2 or 4+ digits. 3 digits is likely to be a thousand separator.
    \d{1,2}|
-   \d{4,8}
+   \d{4}\d*
```

Comment:
:lipstick: No need for a new commit, but for future reference, this should be equivalent: `\d{4,}`